force-app/main/default/permissionsets/Global_Growth_Base_Permission_Set.permissionset-meta.xml

```diff
@@ -499,11 +499,6 @@
         <field>OpportunityLineItem.Attendees_Per_Session__c</field>
         <readable>true</readable>
     </fieldPermissions>
-    <fieldPermissions>
```

Comment:
Can we also delete this field (Billing Type), Number of Sessions, and Attendees per session from the repository